python/pyspark/sql/functions.py

```diff
@@ -1802,10 +1802,10 @@ def from_json(col, schema, options={}):
 @since(2.1)
 def to_json(col, options={}):
     """
-    Converts a column containing a [[StructType]] into a JSON string. Throws an exception,
-    in the case of an unsupported type.
+    Converts a column containing a [[StructType]] or [[ArrayType]] into a JSON string.
+    Throws an exception, in the case of an unsupported type.
```

Comment:
Hm, in that sense, there are many instances, for example, `Returns the double value`, `Returns the positive value` or ``Returns `null` ...``. They return a column. Another example is, `Extracts json object ...`. It does not extract it right there but make a column.